docs/layers/scatterplot-layer.md

```diff
@@ -112,6 +125,36 @@ The rgba color of each object, in `r, g, b, [a]`. Each component is in the 0-255
 * If an array is provided, it is used as the color for all objects.
 * If a function is provided, it is called on each object to retrieve its color.
 
+##### `getFillColor` (Function|Array, optional) ![transition-enabled](https://img.shields.io/badge/transition-enabled-green.svg?style=flat-square")
+
+* Default: `[0, 0, 0, 255]`
+
+The rgba color of each object, in `r, g, b, [a]`. Each component is in the 0-255 range.
+
+* If an array is provided, it is used as the filled color for all objects.
+* If a function is provided, it is called on each object to retrieve its color.
+* If not provided, it falls back to `getColor`.
```

Comment:
`getColor` not documented

Reply:
Explain in the getColor section that it may be overriden if the new accessors are specified.